setup.py

```diff
@@ -275,16 +275,19 @@ def run(self):
 class Webhost(distutils.cmd.Command):
     description = 'Download and setup Azure Functions Web Host.'
     user_options = [
-        ('webhost-version', None,
+        ('webhost-version=', None,
             'A Functions Host version to be downloaded (e.g. 3.0.15278).'),
-        ('webhost-dir', None,
+        ('webhost-dir=', None,
             'A path to the directory where Azure Web Host will be installed.'),
+        ('use-branch=', None,
```

Comment:
use-branch sounds like a boolean field, maybe branch-name?

Reply:
nit: make the comments similar, for example webhost-version you have an e.g. 3.0.15278, it will be nice if you provide e.g for webhost-dir and branch-name/use-branch as well